src/components/PrimaryPanes/Outline.css

```diff
@@ -72,6 +77,10 @@
   white-space: nowrap;
 }
 
+.outline-list > .outline-list__element {
+  padding-left: 0;
+}
+
 .outline-list__element-icon {
   padding-right: 0.4rem;
```

Comment:
So here it would be interesting to:
- use `padding-inline-start` and `padding-inline-end` instead of the padding-left/right properties;
- maybe only keep the space between the icon and the label (it makes more sense to manage the indentation with the list item's padding, IMO).